src/AddTaskButton.js

```diff
@@ -0,0 +1,9 @@
+import React from "react"
+
+function AddTaskButton() {
+  return (
+    <button type="button">Add Task</button>
+  )
+}
+
+export default AddTaskButton
```

Comment:
Above on line 3, you can write `export default function AddTaskButton() { ....` export/define in the same declaration